### DISCUSSION TITLE
How to enable HierarchicalMenu showMore only to child-level facet values independently

### DISCUSSION THREAD

**Discussion by jhaankit** [algolia/instantsearch#5278] [Ideas]
HierarchicalMenu showMore feature works at the parent level and child levels combinedly. 
How to enable it only for child-level facet values independently?

I am referring to the link: https://react-instantsearch.netlify.app/storybook/?path=/story/hierarchicalmenu--playground
When I click the 'show more' button, it opens the facet values for all the selected levels (parent level and child levels combinedly).
I want a feature where I can see only facet values of a specific level and not all.

**Comment by mikedavis77** [comment]
Are you wanting to have a hierarchical menu that only shows you the child levels having selected a level 1 parent item or are you wanting to have the ability to only show a certain number of child items but with a 'show more' button at this child level?

**Reply by jhaankit** [reply]
Actually, I want a hierarchical menu that only shows me the child levels having selected a level 1 parent item, as described in another discussion [here](https://github.com/algolia/react-instantsearch/discussions/3414).
I am also looking for some quick alternatives and could not achieve the ability to show a certain number of child items but with a 'show more' button at this child level.
